explainability-service/src/main/resources/application.properties

```
@@ -31,6 +31,12 @@ quarkus.openshift.mounts.volume.path=/inputs
 quarkus.openshift.pvc-volumes.volume.claim-name=trustyai-service-pvc
 # Dev
 quarkus.kubernetes.deployment-target=openshift
+service.storage-format=memory
+service.data-format=csv
+service.metrics-schedule=5s
+storage.data-filename=data.csv
+storage.data-folder=/inputs
+service.batch-size=5000
```

If this is necessary for dev-mode let's make them for dev using [the profile](https://quarkus.io/guides/config-reference#profiles) :) 
```suggestion
%dev.service.storage-format=memory
%dev.service.data-format=csv
%dev.service.metrics-schedule=5s
%dev.storage.data-filename=data.csv
%dev.storage.data-folder=/inputs
%dev.service.batch-size=5000
```

If we don't like to mix them in the same file it is possible to create another application properties file named `application-dev.properties` (see [this](https://quarkus.io/guides/config-reference#profile-aware-files))